docs/marionette.view.md

```diff
@@ -73,32 +73,43 @@ For more detail on how to render templates, see the
 Marionette is able to manage pre-generated pages - either static or
 server-generated - and treat them as though they were generated from Marionette.
 
-To use the existing page, set the `template` attribute to `false`:
+To use the existing page, set the `el` to match the existing DOM element:
 
 ```javascript
 var Mn = require('backbone.marionette');
 
 var MyView = Mn.View({
-  el: '#base-element',
-  template: false
+  el: '#base-element'
 });
+
+new myView();
+myView.isRendered(); // true if '#base-element` exists
+myView.isAttached(); // true if '#base-element` is in the DOM
 ```
 
 [Live example](https://jsfiddle.net/marionettejs/b2yz38gj/)
 
-Marionette will then attach all the usual
-[`event`](#events-and-callback-methods) and [`ui`](#organizing-your-view)
-handlers to the view using the existing HTML. Though the View has no template,
-you can still listen to the `before:render` and `render` events that will fire
-as usual when `render` is called - or when you execute `region.show(view)`.
 
-### Setting a falsy `template` value
 
-When using an existing page, Marionette explicitly looks for `false` - any other
-falsy value will cause Marionette to raise an error when attempting to render
-the template.
+Marionette will [set the appropriate state of the view](./viewlifecycle.md#views-associated-with-previously-rendered-or-attached-dom).
+
+### Setting a `template` to `false`
+
+Setting the `template` to `false` allows for the view to create all of the bindings and trigger all view events without re-rendering the el of the view. Any other falsy value will throw an exception.
```

Comment:
I'd be tempted to wrap the last sentence in italics: `*Any other falsy value will throw an exception.*`